frontend/src/views/MovieAddScreen.vue

```diff
@@ -121,7 +121,8 @@ export default {
         if (this.image) {
           const data = new FormData();
           data.append('image', this.image);
-          await axios.post(`/api/cinema/movies-${movie.id}-upload-image`, data, imageConfig);
+          await axios.post(`${import.meta.env.VITE_API_URL}/api/cinema/movies-${movie.id}-upload-
+          image/`, data, imageConfig);
```

Comment:
Have you tried to upload image for a movie? I suppose no because you'll get error :)
Try to do it and add screenshot of developer tool